src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -37,10 +40,14 @@
 import seedu.address.model.facilitator.Name;
 import seedu.address.model.facilitator.Office;
 import seedu.address.model.facilitator.Phone;
-import seedu.address.model.modulecode.ModuleCode;
+import seedu.address.model.module.ModuleCode;
 import seedu.address.testutil.FacilitatorBuilder;
 
 public class AddCommandParserTest {
+    private static final String PHONE_EMPTY = " " + PREFIX_PHONE;
```

Comment:
It seems that you never use this anywhere in this test class. You plan to add more tests is it?